src/data_hub/plotting.py

```diff
@@ -24,6 +26,8 @@ def scatter(
     y: Optional[str] = None,
     size: Optional[str] = None,
     hover_callback_code: Optional[str] = None,
+    button_callback_code: Optional[str] = None,
+    label_column: Optional[str] = None,
```

Comment:
these 2 JS files are the only ones that interact with the deeporigin JS API, so allowing us to override it at execution time for faster debug cycles. 

this will eventually be removed